### PR TITLE
feat: add webp to allowed images

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR adds webp to the list of allowable image types in Isomer. To be reviewed in conjunction with PR[#623](https://github.com/isomerpages/isomercms-backend/pull/623) on the isomercms-backend repo.

### CODE REVIEW

File: src/components/MediaCreationModal/MediaCreationModal.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const MediaCreationModal = ({
           id="file-upload"
           accept={
             mediaRoom === "images"
-              ? "image/jpeg, image/png, image/gif, image/svg+xml, image/tiff, image/bmp, image/x-icon"
+              ? "image/jpeg, image/png, image/gif, image/svg+xml, image/tiff, image/bmp, image/x-icon, image/webp"
```
Comment (seaerchin) [inline src/components/MediaCreationModal/MediaCreationModal.jsx:63]: this is fine for now cos there's only 1 call site but perhaps we should shift this to a constant and place this in an appropriate file for easy discovery?

File: src/components/MediaSettingsModal/MediaSettingsModal.jsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ export const MediaSettingsModal = ({
             <p>
               For {mediaRoom} other than
               {mediaRoom === "images"
-                ? ` 'png', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg'`
+                ? ` 'png', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg', 'webp'`
```
Comment (seaerchin) [inline src/components/MediaSettingsModal/MediaSettingsModal.jsx:98]: see above

File: src/components/MediaSettingsModal/MediaSettingsSchema.jsx [hunk 3]
```diff
@@ -38,7 +38,7 @@ export const MediaSettingsSchema = (existingTitlesArray = []) =>
         if (mediaRoom === "images") {
           return schema.test(
             "Special characters found",
-            "Title must end with one of the following extensions: 'png', 'jpeg', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg'",
+            "Title must end with one of the following extensions: 'png', 'jpeg', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg', 'webp'",
```
Comment (seaerchin) [inline src/components/MediaSettingsModal/MediaSettingsSchema.jsx:41]: if we use a named constant, it'll be easier to do such updates

File: src/components/media/MediasSelectModal.jsx [hunk 4]
```diff
@@ -89,7 +89,7 @@ const MediasSelectModal = ({
           <p>
             For {mediaRoom} other than
             {mediaRoom === "images"
-              ? ` 'png', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg'`
+              ? ` 'png', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg', 'webp'`
```
Comment (seaerchin) [inline src/components/media/MediasSelectModal.jsx:92]: similarly for this and other occurrences down below

**Review by seaerchin — COMMENTED** [review]
need to check if we require a fallback image for `webp`; idt this is a high priority for now (we can test with our own gsib) but we should shift to using a constant if there's no good reason otherwise

File: src/components/MediaCreationModal/MediaCreationModal.jsx (hunk 1 above)
Comment (alexanderleegs) [reply src/components/MediaCreationModal/MediaCreationModal.jsx]: Shifted out in https://github.com/isomerpages/isomercms-frontend/pull/1180/commits/1fbc5c845d558299b123ec0568659337e6a15d8f!

File: src/components/MediaSettingsModal/MediaSettingsModal.jsx (hunk 2 above)
Comment (alexanderleegs) [reply src/components/MediaSettingsModal/MediaSettingsModal.jsx]: https://github.com/isomerpages/isomercms-frontend/pull/1180/commits/1fbc5c845d558299b123ec0568659337e6a15d8f

File: src/components/media/MediasSelectModal.jsx (hunk 4 above)
Comment (alexanderleegs) [reply src/components/media/MediasSelectModal.jsx]: https://github.com/isomerpages/isomercms-frontend/pull/1180/commits/1fbc5c845d558299b123ec0568659337e6a15d8f

**Review by seaerchin — APPROVED** [review]
lgtm

**Pull request closed without merging**